mne/viz/epochs.py

```diff
@@ -65,7 +65,7 @@ def plot_epochs_image(epochs, picks=None, sigma=0., vmin=None,
         The scalings of the channel types to be applied for plotting.
         If None, defaults to `scalings=dict(eeg=1e6, grad=1e13, mag=1e15,
         eog=1e6)`.
-    cmap : matplotlib colormap | (colormap, bool) | 'interactive'
+    cmap : None | matplotlib colormap | (colormap, bool) | 'interactive'
```

Comment:
Ok, need to doc this ...
The point is, for the GFP we don't need a diverging cmap (None -> Reds), so I've switched from the RdBu_r default to None.